appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -344,6 +350,42 @@ private static void upgradeComponentProperties(Map<String, JSONValue> componentP
       }  else if (componentType.equals("Notifier")) {
         srcCompVersion = upgradeNotifierProperties(componentProperties, srcCompVersion);
 
+      } else if (componentType.equals("NxtColorSensor")) {
+        srcCompVersion = upgradeNxtColorSensorProperties(componentProperties, srcCompVersion);
+
+      } else if (componentType.equals("NxtDirectCommands")) {
+        srcCompVersion = upgradeNxtDirectCommandsProperties(componentProperties, srcCompVersion);
+
+      } else if (componentType.equals("NxtDrive")) {
+        srcCompVersion = upgradeNxtDriveProperties(componentProperties, srcCompVersion);
+
+      } else if (componentType.equals("NxtLightSensor")) {
+        srcCompVersion = upgradeNxtLightSensorProperties(componentProperties, srcCompVersion);
+
+      } else if (componentType.equals("NxtSoundSensor")) {
+        srcCompVersion = upgradeNxtSoundSensorProperties(componentProperties, srcCompVersion);
+
+      } else if (componentType.equals("NxtTouchSensor")) {
+        srcCompVersion = upgradeNxtTouchSensorProperties(componentProperties, srcCompVersion);
+
+      } else if (componentType.equals("NxtUltrasonicSensor")) {
+        srcCompVersion = upgradeNxtUltrasonicSensorProperties(componentProperties, srcCompVersion);
+
+      } else if (componentType.equals("Ev3Motors")) {
```

Comment:
I'd like to try and keep this function as alphabetized as possible, so please move the Ev3* checks up to the appropriate location.